src/marker.rs

```diff
@@ -698,7 +698,7 @@ impl<'py> Python<'py> {
     /// Gets the Python builtin value `None`.
     #[allow(non_snake_case)] // the Python keyword starts with uppercase
     #[inline]
-    pub fn None(self) -> &'py PyNone {
+    pub fn None(self) -> Borrowed<'py, 'py, PyNone> {
         PyNone::get(self)
     }
```

Comment:
As you might have noticed in the migration guide, we changed `Python::None` to return `&'py PyNone` instead of `PyObject` already since 0.20. I'm not super keen on changing it directly to `Borrowed<PyNone>` as I think that's forcing the new API on users in a breaking change, which we're trying very hard not to go.

I wonder, maybe given all the other changes that are happening in 0.21, perhaps we should just change this back to `PyObject` and we can change it to `Borrowed` in 0.22, 0.23 or later?

That would reduce the amount of work users need to do to migrate to 0.21, which can only be a good thing...

Reply:
... maybe is it useful if I merge this revert quickly? https://github.com/PyO3/pyo3/pull/3794

Reply:
Ok I merged that, if you rebase it should hopefully make the amount of changes needed in this PR a bit simpler!